pkg/integrations/postgres_exporter/postgres_exporter.go

```diff
@@ -85,7 +85,7 @@ func parsePostgresURL(url string) (map[string]string, error) {
 	res := map[string]string{}
 
 	for _, keypair := range strings.Split(raw, " ") {
-		parts := strings.SplitN(keypair, "=", 1)
+		parts := strings.Split(keypair, "=")
```

Comment:
This needs to be SplitN with 2 otherwise a valid `=` in a field from a user will cause >2 splits and trigger the panic.

```suggestion
		parts := strings.SplitN(keypair, "=", 2)
```